### PR TITLE
E-1298 feat(ui): add vendor icons

### PR DESCRIPTION
**Pull request by ifiri**
Had add icon for apple and google play

### CODE REVIEW

**Comment by ifiri** [comment]
@dudiq

File: packages/ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@endpass/ui",
-  "version": "0.19.3",
+  "version": "0.19.5",
```
Comment (dudiq) [inline packages/ui/package.json:3]: it should be 0.19.4

Comment (dudiq) [inline packages/ui/package.json:3]: it should be 0.19.4. increment for one point only

Comment (ifiri) [reply packages/ui/package.json:3]: it should not, because 0.19.4 is in pr of vault
I just prevent version conflict

Comment (dudiq) [reply packages/ui/package.json:3]: then, in near pr of ui, you should update version to 0.19.6.
in npm you did not have 0.19.4 and please use increase numbering for this.
just fix to `0.19.4`

**Comment by dudiq** [comment]
approved

**Comment by kazurus** [comment]
@ifiri :recycle:

**Pull request merged**